netty/src/main/java/io/grpc/transport/netty/Utils.java

```diff
@@ -86,6 +88,68 @@ static ByteBuf toByteBuf(ByteBufAllocator alloc, ByteBuffer source) {
     return buf;
   }
 
+  /**
+   * TODO(nmittler): Move this to the Http2Error enum.
+   *
+   * @param errorCode the HTTP/2 error code.
+   * @return The {@link Http2Error} enum value or {@code null} if not found.
+   */
+  public static Http2Error toHttp2Error(long errorCode) {
+    for (Http2Error error : Http2Error.values()) {
+      if (error.code() == errorCode) {
+        return error;
+      }
+    }
+    return null;
+  }
+
+  /**
+   * Create a {@link Status} from the given HTTP/2 error code.
+   *
+   * @param errorCode the HTTP/2 error code.
+   * @return a {@link Status} representing the given error.
+   */
+  public static Status http2ErrorCodeToStatus(long errorCode) {
+    Http2Error error = toHttp2Error(errorCode);
+    if (error == null) {
+      return Status.UNKNOWN;
+    }
+
+    String msg = "HTTP/2 Error Code: " + error.name();
+    switch (error) {
```

Comment:
This doesn't seem to follow https://github.com/grpc/grpc-common/blob/master/PROTOCOL-HTTP2.md